components/merkledb/src/views/metadata.rs

```diff
@@ -344,6 +345,19 @@ impl<T: RawAccess> IndexesPool<T> {
         self.set_len(len + 1);
         (metadata, is_phantom)
     }
+
+    pub(crate) fn suffixes<K: BinaryKey + ?Sized>(&self, prefix: &str) -> Vec<K::Owned> {
+        let prefix = concat_keys!(prefix, INDEX_NAME_SEPARATOR);
+        let mut keys = Vec::new();
+
+        for (key, _) in self.0.iter::<Vec<u8>, Vec<u8>, Vec<u8>>(&prefix) {
```

Comment:
Bikeshedding/discussion: `Vec<u8>, Vec<u8>, Vec<u8>` looks dirty, maybe provide type aliases for them within the function body?

Reply:
It's possible to remove first Vec<u8>, maybe it won't be so dirty.